libs/ui/atoms/src/lib/Icon/Icon.tsx

```diff
@@ -80,126 +80,139 @@ enum IconType {
 }
 
 interface IIconProps {
+  size?: number | undefined;
+  color?: string | undefined;
+  backgroundColor?: string | undefined;
+  borderRadius?: number | undefined;
+  onPress?: void;
   name: IconType;
+  iconStyle?: object | undefined;
 }
 
-const size = 17;
-
 function Icon(props: IIconProps) {
+  const iconProperties = {
+    size: props.size || 20,
+    color: props.color || '#009688',
+    backgroundColor: props.backgroundColor || '##007AFF',
+    borderRadius: props.borderRadius || 5,
+    onPress: props.onPress || undefined, // @TODO
```

Comment:
I would use Pressable wrapper maybe

Reply:
And onPress is not even implemented :D

Reply:
It has been added here if it ever needs special preservation.